package.json

```diff
@@ -16,7 +16,7 @@
 		"eslint": "^4.11.0",
 		"eslint-plugin-react": "^7.4.0",
 		"eslint-plugin-react-native": "^3.1.0",
-		"jest": "21.2.1",
+		"jest": "^21.2.1",
```

Comment:
You should avoid `^`. Prefer `~` and apply it everywhere in dev deps. In deps, use fixed versions